src/main/scala/com/fulcrumgenomics/sv/AlignedSegment.scala

```diff
@@ -19,13 +19,21 @@ import scala.collection.{BitSet, mutable}
  * @param positiveStrand true if the read was mapped to the positive strand, false otherwise
  * @param cigar          the cigar **in sequencing order**
  * @param range          the genomic range to which this alignment maps
+ * @param left           the records from which this aligned segment originated from and abut the left genomic range of
+ *                       this segment.  May contain multiple if two aligned segments were merged, and be empty if not
+ *                       tracking.
+ * @param right          the records from which this aligned segment originated from and abut the left genomic range of
+ *                       this segment.  May contain multiple if two aligned segments were merged, and be empty if not
+ *                       tracking.
  */
 case class AlignedSegment(origin: SegmentOrigin,
                           private val readStart: Int,
                           private val readEnd: Int,
                           positiveStrand: Boolean,
                           private val cigar: Cigar,
-                          range: GenomicRange) {
+                          range: GenomicRange,
+                          left: Seq[SamRecord] = IndexedSeq.empty,
+                          right: Seq[SamRecord] = IndexedSeq.empty) {
```

Comment:
`IndexedSeq.empty` is actually a function.  Would be best to just store an empty `IndexedSeq` in the companion object, e.g. `val NoReads = IndexedSeq.empty[SamRecord]` and then refer to it here.